src/isaw.theme/isaw/theme/browser/people.py

```diff
@@ -19,19 +32,48 @@ def people(self):
                 'image_url': '{}/@@images/Image/mini'.format(
                     profile.absolute_url()
                 ),
+                'latitude': named_location.get('latitude'),
+                'longitude': named_location.get('longitude'),
             }
             result.append(data)
 
         return result
 
+    def people_json(self):
+        results = self.people(depth=-1)
+        if not results:
+            return ''
+
+        self.request.response.setHeader("Content-Type", "text/json")
+        self.request.response.setHeader(
+                        "Content-Disposition", 'attachment; filename="people-listing.json"'
+                    )
+
+        people = []
```

Comment:
This is a TINY thing, but I got a bit confused by this, I think because of the name "HEADERS". It's more like a list of relevant keys, right? What about:

    relevant = JSON_KEYS.split(',')

    people = [
        {h: record.get(h, '') for h in relevant}
        for record in results
        if record.get('latitude') and record.get('longitude')
    ]

    return json.dumps(people)
    
Maybe I didn't really understand what's happening.

Reply:
thanks @jessesnyder. You understand perfectly. Even in this case it was only a tail of former CSV implementation.
your solution is very elegant. Im'going to commit that!